src/app/core/shared/item-bitstreams-utils.ts

```diff
@@ -0,0 +1,16 @@
+import { Observable } from 'rxjs/internal/Observable';
+import { RemoteData } from '../data/remote-data';
+import { PaginatedList } from '../data/paginated-list';
+import { Bitstream } from './bitstream.model';
+import { map } from 'rxjs/operators';
+import { hasValue, hasValueOperator } from '../../shared/empty.util';
+
+/**
+ * Operator for turning the current page of bitstreams into an array
+ */
+export const toBitstreamsArray = () =>
```

Comment:
Similar to the existing `metadata.utils.ts`, I think this file should be named `item-bitstreams.utils.ts`. 

Also, I think this `toBitstreamsArray` is missing specs.